Там такая ошибка:

```
=========================== short test summary info ============================
FAILED tests/test_copy_from_text_field.py::test_copy_from_text_field - selenium.common.exceptions.StaleElementReferenceException: 
Message: stale element reference: stale element not found
  (Session info: chrome=147.0.7727.101); For documentation on this error, 
please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#staleelementreferenceexception
```

--- a/tests/test_copy_from_text_field.py
+++ b/tests/test_copy_from_text_field.py
@@ -24,7 +24,7 @@ def test_copy_from_text_field(browser):
 
     create_new_pipeline(browser)
 
-    wait.until(ec.element_to_be_clickable((By.CSS_SELECTOR, ".app-jenkins-logo"))).click()
+    wait.until(ec.element_to_be_clickable((By.ID, "jenkins-head-icon"))).click()
     browser.find_element(By.XPATH, "//a[@href='/view/all/newJob']").click()
     wait.until(ec.element_to_be_clickable((By.XPATH, "//input[@id='name']"))).click()
     browser.find_element(By.XPATH, "//input[@id='name']").send_keys("my_second_pipeline")

